keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/ec2.kt

```diff
@@ -16,6 +16,19 @@
 package com.netflix.spinnaker.keel.ec2
 
 import com.netflix.spinnaker.keel.api.ApiVersion
+import com.netflix.spinnaker.keel.api.ec2.ApplicationLoadBalancerSpec
+import com.netflix.spinnaker.keel.api.ec2.ClassicLoadBalancerSpec
+import com.netflix.spinnaker.keel.api.ec2.ClusterSpec
+import com.netflix.spinnaker.keel.api.ec2.SecurityGroupSpec
+import com.netflix.spinnaker.keel.api.plugins.kind
 
 val SPINNAKER_EC2_API_V1 = ApiVersion("ec2")
 const val CLOUD_PROVIDER = "aws"
+
+val CLUSTER_V1 = kind<ClusterSpec>("ec2/cluster@v1")
```

Comment:
I'd name this `EC2_CLUSTER_V1` for consistency with the Titus implementation and added clarity.

Reply:
Yep, I think I'll stick an `EC2_` prefix on all the kinds